README.md

```diff
@@ -19,20 +19,28 @@ interface.
 
 ### Prerequisites
 
-- A working (and up2date) NodeJS installation, with Yarn as the preferred package manager
+- A working (and up2date) NodeJS installation (`v18, lts/hydrogen`), with Yarn as the preferred package manager
 - A working (and up2date) Rust installation including rustc, clippy, and cargo
-- Optional, but strongly advised: a working [pre-commit][pre-commit] installation
+- Optional, but strongly advised: a working [pre-commit][pre-commit] installation. You can find instructions in the pre-commit section on how to get it working.
+
+Don't forget to install global node dependencies:
+
+```shell
+npm install -g yarn typescript
+```
```

Comment:
We shouldn't need those. Also I would be careful with global npm installs. Instead we should link to the installation guides of above tools:

- https://yarnpkg.com/getting-started/install/
- https://rustup.rs/
- https://pre-commit.com/#install